qa/common/infer_util.py

```diff
@@ -153,37 +162,83 @@ def infer_exact(tester, pf, tensor_shape, batch_size,
 
             # create and register shared memory region for inputs and outputs
             if shm_region_names is None:
```

Comment:
Seems like you can just use the code in `else` clause by setting `shm_region_names` to default values if None

Reply:
Done